content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -0,0 +1,140 @@
+  - Once based solely on CPU requests, generating `aws.cost.amortized.cpu.allocated`.
+  - Once based solely on memory requests, generating `aws.cost.amortized.mem.allocated`.
+
+### Understand which metrics provide the best insight into your costs
+
+- If your workloads are CPU-constrained, only using `cpu.allocated` may work for you. 
+- If your workloads are memory-constrained, exclusively using `mem.allocated` might meet your use case.
+- If you have a mix of CPU and memory-intensive workloads, or you simply want a consistent way to visualize the costs of considering all resources, you can use formulas and functions to create your own equation to combine CPU and memory costs. For example, for a 50/50 split, you can plot:
+
+$$\text"a " = \text" sum(aws.cost.amortized.cpu.allocated) by {team}"$$
+
+$$\text"b" = \text"sum(aws.cost.amortized.mem.allocated) by {team}"$$
+
+$$\text"mixed_cost" = (0.5 * \text"a") + (0.5 * \text"b")$$
+
+As long as the multipliers are positive and sum to 1, the formula will generate coherent costs.
+
+**Note**: For costs that are not related to containers on EC2, all three metrics are equal:
+  - `aws.cost.amortized` 
+  - `aws.cost.amortized.cpu.allocated`
+  - `aws.cost.amortized.mem.allocated`
+
+## Understanding cluster idle cost
+
+Cluster idle cost is the cost of CPU or memory resources that are not reserved by any workloads.
+
+You can visualize this cost at the Kubernetes node, ECS host, or cluster level, using the `is_cluster_idle` tag. All costs allocated to scheduled pods or tasks have `is_cluster_idle: N/A`, while all compute costs not assigned to any pods or tasks have the tag `is_cluster_idle: true`.
+
+Here is a sample query that displays all Kubernetes cluster cost, broken down by cluster name and cluster idle cost:
+   
+`sum:aws.cost.amortized.cpu.allocated{is_kubernetes:true} by {kube_cluster_name, is_cluster_idle}`
+
+  {{< img src="/cloud_cost/container_cost_allocation/cost_allocation_editor_config.png" alt="Example configuration for mixed resources with break down by cluster name and cluster idle cost tags" style="width:100%;" >}}
+
+## Tags
+
+Datadog applies additional tags to cost metrics where container costs are allocated.
+
+### Kubernetes
+
+In addition to Kubernetes pod and Kubernetes node tags, the following out of the box tags are applied to cost metrics:
+
+| Out of the box tag  |  Description |
+| ---                 | ------------ | 
+| `kube_cluster_name` | The name of the kubernetes cluster. |
+| `is_kubernetes`     | All EC2 compute costs associated with running Kubernetes nodes. |
+| `is_cluster_idle`   | The cost of unreserved CPU or memory on Kubernetes nodes. |
+
+### ECS
+
+In addition to ECS task tags, the following out of the box tags are applied to cost metrics. **Note**: most tags from ECS container (excluding `container_name`) are applied.
```

Comment:
Should `**Note**: most tags from ECS container (excluding `container_name`) are applied.` be `**Note**: most tags from ECS containers (excluding `container_name`) are applied.` ?

Reply:
```suggestion
In addition to ECS task tags, the following out of the box tags are applied to cost metrics. **Note**: most tags from ECS containers (excluding `container_name`) are applied.
```